.azure/reusable-build.yml

```diff
@@ -35,7 +35,9 @@ parameters:
   - name: configurations
     type: object
     default: ["Debug", "Release"]
-
+  - name: platform
+    type: object
+    default: ["x64"]
```

Comment:
personally I'd prefer not having a default and making it a required parameter, but approving anyway